File: sdks/java/src/main/java/com/boltprediction/sdk/model/CmmsWebhookResponse.java
package com.boltprediction.sdk.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import java.util.List;
import java.util.Map;
import java.time.OffsetDateTime;

/** CMMS Webhook响应 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class CmmsWebhookResponse {

    @JsonProperty("success")
    private Boolean success;

    @JsonProperty("message")
    private String message;

    @JsonProperty("processed_count")
    private Object processedCount;

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Object getProcessedCount() {
        return processedCount;
    }

    public void setProcessedCount(Object processedCount) {
        this.processedCount = processedCount;
    }

}

File: sdks/python/bolt_prediction_sdk/models/cmms_webhook_response.py
"""
CmmsWebhookResponse 模型
"""

from typing import Optional, Any, Dict, List
from datetime import datetime
from pydantic import Field

from .base import SDKBaseModel


class CmmsWebhookResponse(SDKBaseModel):
    """CMMS Webhook响应"""

    success: bool = Field()
    message: str = Field()
    processed_count: Optional[Any] = Field(default=0)
